xlators/protocol/auth/addr/src/addr.c

```diff
@@ -85,6 +94,7 @@ compare_addr_and_update(char *option_str, char *peer_addr, char *subvol,
 
     ret = -1;
 out:
+    GF_FREE(hostname);
```

Comment:
```suggestion
    if (hostname)
        GF_FREE(hostname);
```

Reply:
I don;t think it is required, GF_FREE already check if ptr is NULL or not.